### DISCUSSION TITLE
Easier 1password passkey implement

### DISCUSSION THREAD

**Discussion by T-Mwangi** [simple-login/app#1771] [Feature Requests]
I am having a slight annoyance with using the security key method for authentication currently. I am using 1passwords passkey feature, and while I can successfully use this as my security key, login in has created extra steps. you have chosen to default to using windows security to authenticate security keys, but in doing so, "breaks" 1passwords attempts to log me in. currently to login using 1passwords passkey feature, I must completely cancel the windows security choice, before i can then click "use security key", to then login. I'm hoping to find a way to have 1passwords passkey log me in immediately, without having to go through windows security.

**Comment by fm** [comment]
I don't think that's a SimpleLogin or even a 1Password issue. I believe the OS/first-party implementations always get first dibs. Broadly speaking, the passkey provider APIs are still in flux at the moment.

**Reply by T-Mwangi** [reply]
I just re-tested the setup for registering a passkey, and now I don't get the windows setup pop-up, but now SimpleLogin seems to setup though the browser, which therefore interacts with 1password, rather than before (where it went directly to the OS Implementation). 

I'm pretty sure the key registration page is also different, so I think that this was a feature SimpleLogin updated between when I pointed out the issue, and now when re-testing. 

I'm glad that pointing out this issue ultimately led to the change, and better implementation of passkeys.
